app/(storyblok)/[[...slug]]/not-found.tsx

```diff
@@ -34,30 +36,24 @@ storyblokInit({
  * Make sure to not export the below functions otherwise there will be a typescript error
  * https://github.com/vercel/next.js/discussions/48724
  */
-async function getStoryData(slug = 'page-not-found') {
-  const activeEnv = process.env.NODE_ENV || 'development';
+async function getStoryData(slug = 'momentum/page-not-found') {
+  const isProd = isProduction();
   const storyblokApi: StoryblokClient = getStoryblokApi();
   const sbParams: ISbStoriesParams = {
-    version: activeEnv === 'development' ? 'draft' : 'published',
-    cv: activeEnv === 'development' ? Date.now() : undefined,
+    version: isProd ? 'published' : 'draft',
+    cv: Date.now(),
```

Comment:
`cv`: Always passing a date here now. This causes cache invalidation (or skipping) and will always fetch the latest. Since we running builds, the latest is what we want and I am not worried about skipping the cache on the SB API.

Reply:
If they're to be trusted, the examples in the docs suggest `cv` is seconds since epoch instead of milliseconds. I guess it doesn't really matter if it interprets it as now or 50,000 years in the future though. 😆

Reply:
That is helpful information. Thank you @ericbakenhus